cmd/operator/main.go

```diff
@@ -218,6 +219,14 @@ func Main() int {
 
 	logger.Log("msg", fmt.Sprintf("Starting Prometheus Operator version '%v'.", version.Version))
 
+	if deprecatedConfigReloaderImage != "" {
+		level.Warn(logger).Log(
+			"msg", "'--config-reloader-image' flag is ignored, only '--prometheus-config-reloader' is used",
```

Comment:
If we are deprecating this right now, why are we ignoring it, should there not be a grace period of one release?

Reply:
I don't see anything wrong here since features provided by both config reloaders are now integrated into the one which is included in this repo. With this, we are not deprecating any features.

Reply:
Interesting question though, I also lean towards the fact that we are not breaking the API (cli options) but change the functionality underneath.

Reply:
What @paulfantom said: we keep the CLI flag to not break existing deployments.
One caveat though: you won't be able to mix this new version of `prometheus-operator` with a previous version of `prometheus-config-reloader` because the former expects new CLI flags from the latter (and also the older `prometheus-config-reloader` versions have [bugs](https://github.com/prometheus-operator/prometheus-operator/pull/3457#issuecomment-688689962) when watching directories).